src/services/profiles/profiles.ts

```diff
@@ -0,0 +1,100 @@
+import type { HttpInstance } from '../../types';
+import { AffectedRecords, PagedResult } from '../types';
+import { RQLString } from '../../rql';
+import { Profile, ProfileCreation, Comorbidities, Impediments } from './types';
+
+export default (client, httpAuth: HttpInstance) => ({
+  /**
+   * Get a list of profiles
+   * Permission | Scope | Effect
+   * - | - | -
+   * none | | View your profile
+   * none | `staff enlistment` | View all the profiles of the group
+   * `VIEW_PATIENTS` | `global` | View all profiles
+   *
+   * @param rql an optional rql string
+   * @returns PagedResult<Profile>
+   */
+  async find(options?: { rql?: RQLString }): Promise<PagedResult<Profile>> {
```

Comment:
Add `findById` and `findFirst`

Reply:
ah good spot! I also added these now to the payments services.